.github/workflows/todo-to-issue.yml

```diff
@@ -31,7 +31,6 @@ jobs:
         with:
           AUTO_ASSIGN: "true"
           INSERT_ISSUE_URLS: "true"
-          CLOSE_ISSUES: "true"
```

Comment:
Nit: Use `CLOSE_ISSUES: "false"` instead. Even though that's the default, it would make this behavior more discoverable to readers.

Reply:
Done